server/routes/prisonerReport/prisonerReportPage.ts

```diff
@@ -110,6 +112,7 @@ export default class prisonerReportRoutes {
       offences,
       statementEditable: !this.pageOptions.isReviewerView(),
       ...prisonerReportVariables,
+      readOnly: ['ACCEPTED', 'REJECTED'].includes(reportedAdjudication.reportedAdjudication.status),
```

Comment:
Maybe we stick these in an enum - that mirrors the enum on the API. I might have a look in the stuff I'm doing